eras/conway/impl/src/Cardano/Ledger/Conway/Governance/Procedures.hs

```diff
@@ -301,7 +302,13 @@ data GovernanceAction era
   | HardForkInitiation !ProtVer
   | TreasuryWithdrawals !(Map (Credential 'Staking (EraCrypto era)) Coin)
   | NoConfidence
-  | NewCommittee !(Set (KeyHash 'Voting (EraCrypto era))) !Rational
+  | NewCommittee
+      -- | Old committee
+      !(Set (Credential 'CommitteeColdKey (EraCrypto era)))
+      -- | New Committee with expiration epoch
+      !(Map (Credential 'CommitteeColdKey (EraCrypto era)) EpochNo)
+      -- | Quorum
```

Comment:
I think haddock will choke otherwise
```suggestion
      -- Old committee
      !(Set (Credential 'CommitteeColdKey (EraCrypto era)))
      -- New Committee with expiration epoch
      !(Map (Credential 'CommitteeColdKey (EraCrypto era)) EpochNo)
      -- Quorum
```

Reply:
Nevermind, I was totally wrong. Looks like haddock is capable of handling haddock like this nowadays. Which is great!